src/ActionModal/components/ActionModal.vue

```diff
@@ -486,6 +486,7 @@ export default {
         url: this.network.api_url,
         networkId: this.network.id,
         chainId: this.network.chain_id,
+        networkType: this.network.network_type,
```

Comment:
How did you get this to work without having `network_type` in the Network schema? (I just added it to your PR)
O.o

Reply:
Magic xD